scripts/pi-hole/php/debug.php

```diff
@@ -49,12 +49,21 @@ function echoEvent($datatext) {
     }
 }
 
+// Execute "pihole" using Web option
```

Comment:
Could we use `pihole_execute` here?

Reply:
No need, just asking if using the wrapper is possible to get a consistent call of `pihole`